tests/valgrind/potential-memleaks.c

```diff
@@ -100,7 +100,8 @@ pl_workthread_strerror_localtime(void * cookie)
 	(void)cookie; /* UNUSED */
 	(void)str; /* UNUSED */
 
-	time(&now);
+	if (time(&now))
```

Comment:
What's the return value of `time(3)`?

Reply:
"any of the reasons described in gettimeofday(2)".

... and yes, I should fix the commit message.

Reply:
... ah, `man 3 time` does go on to specify that neither C99 nor POSIX.1 requires time() to see errno on failure.

Reply:
That too, but I said return value, not errno.